eng/versioning/external_dependencies.txt

```diff
@@ -183,8 +183,8 @@ com.microsoft.azure:azure-storage;8.0.0
 com.microsoft.azure:msal4j;1.11.0
 com.microsoft.azure:msal4j-persistence-extension;1.1.0
 com.sun.activation:jakarta.activation;1.2.2
-io.opentelemetry:opentelemetry-api;1.0.0
-io.opentelemetry:opentelemetry-sdk;1.0.0
+io.opentelemetry:opentelemetry-api;1.7.0
+io.opentelemetry:opentelemetry-sdk;1.7.0
```

Comment:
@lmolkova is it ok to bump the min OpenTelemetry API and SDK level globally? and if so, is it ok for us to go ahead and bump it to the latest OpenTelemetry version `1.10.0`?

Reply:
yes, it will be fine. We probably *should* do less of it after the opentelemetry plugin and exporter GA, but now it's totally fine to do.